src/scripts/main.js

```diff
@@ -1,3 +1,14 @@
 'use strict';
 
-// write code here
+const tree = document.querySelector('.tree');
```

Comment:
Ensure that the element with the class 'tree' exists in the DOM to avoid potential runtime errors.